test/index.test.js

```diff
@@ -217,6 +217,14 @@ function runTests (Obj, name) {
       test('should not allow local variables to override systemvars', () => {
         expect(envTest({ path: envSystemvars, systemvars: true })['process.env.PATH2'] !== '""').toEqual(true)
       })
+
+      test('Should give the highest priority for the system variables', () => {
+        process.env.TEST = 'production'
```

Comment:
A small nit-picky thing, but can we name this variable something more outlandish? `TEST` is pretty generic.

I'll let you be creative as it is your PR, but if you are looking for inspiration you can name it `MATT_MADE_ME_DO_THIS`.

Reply:
as far as i can tell this is on purpose as we want to override the defined default variables.
Renaming this would result in renaming all env variables `TEST`